src/app/src/main/resources/config.properties

```diff
@@ -30,5 +30,6 @@ oauth2_client_id=fakeId
 oauth2_client_secret=fakeSecret
 oauth2_client_url=fakeUrl
 oauth2_client_audience=naomi
-oauth2_client_adr_url=http://localhost:5000
-oauth2_login_method=false
\ No newline at end of file
+oauth2_client_adr_server_url=http://localhost:5000
+oauth2_login_method=false
```

Comment:
I think this is a bit confusing since we're using oauth2 to do the hint login, but this just refers to using oauth2 to pass ADR credentials for fetching resources? So maybe `oauth2_adr_resources` or something similar might be clearer?

Reply:
Sorry, I've just realised that this was the config key we were already using for supporting all Auth0 login, as opposed to legacy HINT login, right? So we're assuming that SSO for getting ADR resources is the method to use so long as Auth0 access is what we're using to log in at all? Sorry, I'd forgotten we were still supporting legacy login,